bundles/CoreBundle/src/EventListener/Frontend/ContentTemplateListener.php

```diff
@@ -83,14 +82,16 @@ public function onKernelView(ViewEvent $event): void
         $parameters = $this->resolveParameters($event->controllerArgumentsEvent, $attribute->vars);
         $status = 200;
 
-        foreach ($parameters as $k => $v) {
-            if (!$v instanceof FormInterface) {
-                continue;
+        if (class_exists('Symfony\\Component\\Form\\FormInterface')) {
```

Comment:
```suggestion
        if (interface_exists('Symfony\\Component\\Form\\FormInterface')) {
```
this fixes that, nice catch @aryaantony92